index.js

```diff
@@ -21,7 +21,7 @@ module.exports = robot => {
         'feature',
         'bugfix',
         'docs',
-        'discussion',
```

Comment:
I'm not a fan of the `discussion` label, so this more specific label seems fine to me.  It's worth noting that we are also going to be adding the `infra` and `dependencies` labels in the near future.